### ISSUE TITLE
Error saving numishare configuration file?

### ISSUE THREAD

**Issue by jtstratford** [ewg118/numishare#94]
We were able to import records following your instructions, so thanks for that.

We have one unrelated issue using numishare.  When logged in to the administrative account we can create collections and save them without an issue but if we try to "modify settings" we get an "Error writing to numishare configuration file."  We assume it's some kind of permissions issue--can you advise where we might look and what you think the permissions should be to fix this?

Thanks for all your help.

**Comment by ewg118** [comment]
It seems more likely there's a conflict between what is in the config XML document and the validation enforced on the settings page (vs. the admin panel). Try loading the settings page and then saving it (without making changes). If there's an error, please copy and paste your config file (which you can access through the eXist-db REST interface at http://servername:8080/exist/rest/db/projectname/config.xml) and I can try to find where the error is in the validation. If the error only occurs after making some changes, please let me know what steps you took. I'm 95% sure this is a validation error, or else the admin panel would have errored when trying to create the new collection.

**Comment by jtstratford** [comment]
Yes, if we load the settings page and make no changes we still get the error.  We have a couple of collections.  Here is the config.xml from one of them.  Any insights appreciated.
[config.xml.txt](https://github.com/ewg118/numishare/files/4320485/config.xml.txt)

**Comment by ewg118** [comment]
The copyrightHolder is no longer required, so it has been removed as an input from the "create a new collection" page in the Admin panel, but the binding was still active on the Settings page. A line in the 150s in xforms/settings.xhtml can be removed:

`<xforms:bind nodeset="copyrightHolder" required="true()"/>`

I have committed the change to Github. Try removing this line and try saving the settings page again. It may take some trial and error.
